envisionpy/processor_network/Bandstructure3DNetworkHandler.py

```diff
@@ -0,0 +1,261 @@
+# Name: Bandstructure3DNetworkHandler 
+import sys,os,inspect
+import inviwopy as ivw
+import inviwopy.glm as glm
+import h5py
+import numpy as np
+
+app = ivw.app
+network = app.network
+
+factory = app.processorFactory
```

Comment:
use the inviwoApp that is sent as an argument. Otherwise you will have 2 instances